### ISSUE TITLE
In-App Downloader does not download files to Android device

### ISSUE THREAD

**Issue by Pentaphon** [devgianlu/Aria2App#54]
Shouldn't the in-app downloader recently added to Aria2app make it so that files can be downloaded without configuring anything else? Everything I try to download gets Error #16 Download Aborted. I'd like to be able to download files to my phone just by selecting the in-app downloader and adding URLs, but that doesn't seem to work on a clean installation of 5.0.1.

**Comment by devgianlu** [comment]
The downloads error out probably because the app doesn't have the storage write permission (bug). As a workaround try to grant it manually from the settings.

**Comment by Pentaphon** [comment]
@devgianlu You're right! It didn't ask me for the storage permission! I granted it manually and now it's working! Thanks.
